packages/http-client-csharp/emitter/src/lib/client-model-builder.ts

```diff
@@ -31,10 +33,22 @@ export function createModel(sdkContext: SdkContext<NetEmitterOptions>): CodeMode
 
   const sdkPackage = sdkContext.sdkPackage;
 
-  const modelMap = new Map<string, InputModelType>();
-  const enumMap = new Map<string, InputEnumType>();
-
-  navigateModels(sdkContext, modelMap, enumMap);
+  const typeCache: TypeCache = {
+    cache: new Map<SdkType, InputType>(),
```

Comment:
consider to rename `cache` to `types` to be consistent with `models` and distinguish from `typeCache` itself

Reply:
or we can use `SdkTypeToInpuTypeMap` as name. It is a little complex but it clearly give out the content and purpose of the map